src/test/java/executor/service/model/StepDTOTest.java

```diff
@@ -0,0 +1,84 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StepDTOTest {
+
+    private final String action = "click";
+    private final String value = "body";
+    private StepDTO expectedStepDTO;
+
+    @BeforeEach
+    public void setUp() {
+
+        expectedStepDTO = new StepDTO(action, value);
+
+    }
+
+    @Test
+    void testEqualOrNot() {
```

Comment:
let's use the same code style for all test
```suggestion
    public void testEqualOrNot() {
```

Reply:
Sorry, I was inattentive, I made edits